config/defaults.conf

```diff
@@ -234,18 +234,17 @@ bert_fine_tune = 0  // If false, BERT transformer weights will
                     // lowering the batch size if set to true.
 
 // Sentence Encoder //
-sent_enc = rnn  // Type of sentence encoder: 'bow', 'rnn' (for LSTM), or 'transformer'
-                // set to 'null' to do nothing but forward the skip connection.
+sent_enc = rnn  // Type of sentence encoder: 'bow', 'rnn' (for LSTM),, or 'transformer'
+                // set to 'null' to feed the output of the pretrained encoder into the task specific module.
                 // Note: Transformer has no known issues, but hasn't performed well, so there may be bugs.
-                // Note: 'bow' just skips the encoder step and passes the word representations to the task model, s
+                // Note: 'bow' just skips the encoder step and passes the word representations to the task model,
                 //   so it is possible to combine attention or max pooling with the 'bow' encoder.
 bidirectional = 1  // If true, the RNN encoder should be bidirectional.
 d_hid = 1024  // Hidden dimension size (usually num_heads * d_proj for transformer)
 n_layers_enc = 2  // Number of encoder layers. Usually 8–12 for Transformer.
 skip_embs = 1  // If true, concatenate the encoder's input (ELMo or embeddings) with the encoder's output.
-sep_embs_for_skip = 0  // Whether the skip embedding uses the same embedder object as the original embedding (before skip).
-                       // Only makes a difference if we are using ELMo weights, where it allows the four tuned ELMo scalars
-                       // to vary separately for each target task.
+// Not supported for BERT and OpenAI GPT.
+sep_elmo_embs_for_skip = 0  // Whether the skip embedding uses the same embedder object as the original embedding (before skip), used only for ELMO
```

Comment:
Leave in the comment about scalars—I think that's helpful context.

Reply:
I think this setting name is kind of unclear. Could we switch to something like `elmo_skip_connection_per_task` or something that conveys that this will have a different set of weights for each task